sw/airborne/modules/display/max7456.c

```diff
@@ -263,6 +393,75 @@ static char ascii_to_osd_c(char c)
 #endif
 }
 
+static void calc_flight_time_left(void)
+{
+  float current_amps = 0;
+  float horizontal_speed = 0;
+  static float bat_capacity_used = 0;
+
+  current_amps = electrical.current;
+  bat_capacity_used += (current_amps*1000.) / (3600. * (float)MAX7456_PERIODIC_FREQ);
+  bat_capacity_left = (float)BAT_CAPACITY - bat_capacity_used;
+  if(bat_capacity_left < 0){ bat_capacity_left = 0; }
+  horizontal_speed = state.h_speed_norm_f;
```

Comment:
again, use getter function, same for later occurrences

Reply:
FIXED